product_docs/docs/biganimal/release/using_cluster/tagging/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Tagging BigAnimal resources
+---
+
+BigAnimal provides a shared tags system that allows you to assign and manage tags for resources across different resource types.
+
+The tags are assigned to the following resource types:
+
+- Project 
+- Cluster
+
+Key features of shared tags system are:
+
+- **Shared tags** &mdash; You can [create shared tags](./create_and_manage_tags.mdx) that are accessible and applicable to all the resource types. Shared tags provides a standard way to categorize and organize resources. Tags are scoped to the organizations and shared among all users in the organization. The access to tags is controlled by existing permission system.
```

Comment:
we just said applicable to Project and Cluster, and here , we say that it is applicable to all resource type

Reply:
Yes, so first, we listed all resource types supported (Project and Cluster) and then referred to all the resource types. Do you think this makes sense - " applicable to all the supported resource types"?